### PR TITLE
Fix copy/paste for documents containing a `DamFileDownloadLinkBlock`

### PR DESCRIPTION
**Pull request by thomasdax98**
### Previously

The linked file wasn't copied to the target scope when copying a document containing a `DamFileDownloadLinkBlock` between scopes

### Now

The file referenced by the `DamFileDownloadLinkBlock`  is correctly copied to the target scope

## Changeset

-   [x] I have verified if my change requires a changeset

## Related tasks and documents

https://vivid-planet.atlassian.net/browse/COM-1058

### CODE REVIEW

**Review by nsams — APPROVED** [review]

File: packages/api/cms-api/src/dam/blocks/dam-file-download-link-block-transformer.service.ts [hunk 1]
```diff
@@ -34,13 +34,15 @@ export class DamFileDownloadLinkBlockTransformerService implements BlockTransfor
                 name: file.name,
                 fileUrl: await this.filesService.createFileUrl(file, { previewDamUrls, relativeDamUrls }),
                 size: Number(file.size),
+                scope: file.scope,
```
Comment (johnnyomair) [inline packages/api/cms-api/src/dam/blocks/dam-file-download-link-block-transformer.service.ts:37]: Why is this needed? And if it's needed, it probably should be in the block meta and handled in `output2State` correctly.

Comment (thomasdax98) [reply packages/api/cms-api/src/dam/blocks/dam-file-download-link-block-transformer.service.ts:37]: It's necessary because when copying a page to another scope, all files on the page are also copied if they do not exist in the scope. For this check, the file scope is necessary (see https://github.com/vivid-planet/comet/blob/356ba1737f017325525823bbffba640910bea56f/packages/admin/cms-admin/src/pages/pageTree/useCopyPastePages/sendPages.tsx#L149-L150)

I did it the same way it's done for the PixelImageBlock (https://github.com/vivid-planet/comet/blob/e1592759458bb9c9b47b1eb3e997bf9a0cb40cc8/packages/api/cms-api/src/dam/blocks/pixel-image-block-transformer.service.ts#L62)

**Review by johnnyomair — APPROVED** [review]

**Review by johnnyomair — APPROVED** [review]
